Documentation/Contributing/documentation.md

```diff
@@ -35,9 +35,10 @@ When previewing, now you can navigate your browser to [http://127.0.0.1:8000/](h
 
     Please make sure that your Python binary path is included in your `PATH`.
 
-## Running helm-docs
-[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, you need to run helm-docs manually, and check in the resulting autogenerated md files at the path `/Documentation/Helm-Charts`
+## making docs
+[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, the developer needs to run  `make docs` and check in the resulting autogenerated md files under  the path `/Documentation/Helm-Charts`. 
+
+running  `make docs` will do everything that is needed including invocation of helm-docs.
```

Comment:
Do we need lines 41-43? They seem like details that only someone looking at makefiles needs to know. IMO the developer guide doesn't need these details for normal usage.

Reply:
@travisn  wrote:
> Do we need lines 41-43? They seem like details that only someone looking at makefiles needs to know. IMO the developer guide doesn't need these details for normal usage.

I fully agree. removed